### DISCUSSION TITLE
capstone ML scooty

### DISCUSSION THREAD

**Discussion by tatapinandhika** [teamalgoritma/community#145] [Q&A] [answered]
saya sudah mencoba melakukan floordate berdasarkan unit=hours, tetapi kenapa waktunya malah tidak muncul ya?. Dan satu lagi, bagaimana cara membuat kolom demands, agar bisa menampung jumlah orderan di waktu dan dan daerah yang sama. makasi bantuannya. 

<img width="1089" alt="Screen Shot 2022-06-21 at 09 50 45" src="https://user-images.githubusercontent.com/101564223/174700124-a9244030-d887-4d74-8fbb-d79f081fa02a.png">

**Comment by kevbow** [comment, marked as answer]
Halo @tatapinandhika,

Penulisan kode di dalam mutate pada piping tidak usah menggunakan keterangan dataframe karena sudah mengacu pada dataframe "scooty" yang Bapak tuliskan di awal. Bapak bisa coba untuk mengubah kode yang ditulis menjadi:

```
scooty_clean <- scooty %>% 
  select(start_time, src_sub_area) %>% 
  mutate(start_time = ymd_hms(start_time)) %>% 
  mutate(start_time = floor_date(start_time, unit = "hours"))
```

Demikian yang dapat kami sampaikan, apabila jawaban ini sesuai dengan pertanyaan Anda harap ditandai dengan "Mark as answer".
